docs/pages/autorc.md

```diff
@@ -132,16 +181,41 @@ You can customize everything about a label
 }
 ```
 
+#### Release Type: `none`
+
+A label with the `none` release type will not create a release when merged.
+If paired with a SEMVER label, the release is not skipped.
+
+```json
+{
+  "labels": [
+    {
+      "name": "documentation",
+      "releaseType": "none"
+    }
+  ]
+}
+```
+
 #### Changelog Titles
 
-By default auto will create sections in the changelog for the following labels.
+Each PR included in the release will be matched to the first label defined in the config with a `changelogTitle` in priority order of the `releaseType` (major, minor, patch, then all others) and included as an entry within that section
```

Comment:
@hipstersmoothie 
I agree that the wording can be improved...

One piece of info that we may lose by removing that wording is the following scenario:

my config
```
{
  "labels": [
    { "releaseType": "none", "name": "foo", "changelogTitle": "Foo" },
    { "releaseType": "minor", "name": "bar", "changelogTitle": "Bar" }
  ]
}
```

if a PR has both `foo` and `bar` labels, then technically, the first label in the config is `foo`, but the PR would be assigned to the `bar` section.

This likely will not be a common use case, but wondering if it is worth keeping the info in the docs in a different format for completeness.

What are your thoughts?

Reply:
> Ah I see. I just wanted to break up the sentence a little bit. I agree with
you we should keep the original wording maybe a little clearer

Definitely open to suggestions about how to improve the wording.

How about rewording to something like below?

---

#### Changelog Titles

Each PR included in the release will be assigned to a label section based upon the matching label with the highest `releaseType` that has a `changelogTitle`.
- Priority order of `releaseType` from highest to lowest: major, minor, patch, and then all others
- If a PR has multiple labels of the same `releaseType`, then the PR is assigned based upon the label that is assigned first in the config

By default auto will create sections in the changelog for the following labels:

- major
- minor
- patch
- internal
- documentation

For example:

- Using the default config, if a given PR has the labels `minor` and `internal`, then it will be included in the `minor` label section
- Using the default config, if a given PR has the labels `documentation` and `internal`, then it will be included in the `internal` label section

Reply:
I like it!

Reply:
should be addressed in: 0d286f9

Reply:
should be addressed in: 0d286f9